.eslintrc.js

```diff
@@ -29,7 +29,13 @@ module.exports = {
     }],
     'max-statements': ['error', 24],
     'no-console': 'off',
-    'comma-dangle': ['error', 'always-multiline'],
+    'comma-dangle': ['error', {
+      arrays: 'always-multiline',
+      objects: 'always-multiline',
+      imports: 'always-multiline',
+      exports: 'always-multiline',
+      functions: 'only-multiline'
```

Comment:
any reason for only-multiline here?

Reply:
Didn't want to introduce unrelated code style changes in this PR. comma-dangle used to ignore functions so we have a mix of both.